src/main/java/games/strategy/triplea/attachments/TriggerAttachment.java

```diff
@@ -2464,7 +2464,7 @@ public static void triggerUnitPlacement(final Set<TriggerAttachment> satisfiedTr
       for (final PlayerID aPlayer : t.getPlayers()) {
         for (final Territory ter : t.getPlacement().keySet()) {
           for (int i = 0; i < eachMultiple; ++i) {
-            placeUnits(t, ter, t.getPlacement().get(ter), aPlayer, data, aBridge);
```

Comment:
Local variable `data` (declared above on line 2448) is now unused.  Leaving it in place to preserve any possible side effects caused by the expression `aBridge.getData()`.